app/jobs/full_text_search/update_issue_content_job.rb

```diff
@@ -47,7 +52,14 @@ def create_content(issue_id, excludes: [])
                 .reject {|j| j.notes.blank? || excludes.include?(j.id) }
                 .sort_by(&:id)
                 .map(&:notes)
+      attachment_contents = issue.attachments
+                              .order(:id)
+                              .reject {|a| excludes.include?(a.id) }
```

Comment:
Is this needed?
It seems that the destroyed attachment record doesn't exist when we execute this.

Reply:
Ahhh I think we don't need it.